### ISSUE TITLE
Issue with accountDeletionDate in CurrentUser.yaml

### ISSUE THREAD

**Issue by Rexios80** [vrchatapi/specification#11]
```
accountDeletionDate:
    oneOf:
      - $ref: ./Date.yaml
      - nullable: true
```

I think something is wrong with this piece but I don't know enough about the OpenAPI specification to fix it.

Here are some links I've found regarding this:
https://github.com/Redocly/redoc/issues/1303
https://stackoverflow.com/questions/40920441/how-to-specify-a-property-can-be-null-or-a-reference-with-swagger

I tried doing some of the things in those links but I ended up just making it worse

**Comment by Rexios80** [comment]
Another note on this is I'm pretty sure OpenAPI 3.1 supports this, but I tried upgrading to it and everything broke.

**Comment by Foorack** [comment]
As we are still on 3.0 we have to fix this with the:
```yaml
type: string
nullable: true
```
method. Once we go to 3.1 we need to change this to:
```yaml
oneOf:
  - type: 'null'   # Note the quotes around 'null'
  - type: string
```
or:
```yaml
type:
  - 'null'   # Note the quotes around 'null'
  - string
```
Although 3.1 is currently so incredibly unsupported by everything it will likely be quite a while until that transition happens...

**Comment by Rexios80** [comment]
This did not generate code with the date as nullable

**Comment by Foorack** [comment]
Question is if the generator can do that. This is the only variable in the spec right now that tends to be present with value `null`, rather than simply not being present like all other variables.

**Comment by Foorack** [comment]
@Rexios80 How does it do with variables which are not required? Does it mark those as nullable? Would marking `accountDeletionDate` as non-required fix the generation?

**Comment by Rexios80** [comment]
Yes but it is technically required. I think the only way to truly fix this (assuming it actually works) would be openapi 3.1.

**Comment by Foorack** [comment]
Fixed by #19
